salt/metalk8s/saltmaster/files/salt-master-pod.yaml.j2

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: salt-master
+  namespace: kube-system
+  labels:
+    app: salt-master
+    app.kubernetes.io/name: salt-master
+    app.kubernetes.io/version: '{{ salt_master_version }}'
+    app.kubernetes.io/component: salt
+    heritage: metalk8s
+    app.kubernertes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
+  hostNetwork: true
+  priorityClassName: system-cluster-critical
+  securityContext:
+    runAsUser: 0
+    fsGroup: 0
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+  containers:
+    - name: salt-master
+      image: docker.io/library/{{salt_master_image}}:{{salt_master_version}}
+      imagePullPolicy: Never
+      ports:
+        - name: master1
```

Comment:
`publisher` (https://docs.saltstack.com/en/getstarted/system/communication.html)

Reply:
It still says `master1` and `master2`?

Reply:
@sayf-eddine-scality is this solved?